libvuln/opts.go

```diff
@@ -16,29 +17,70 @@ import (
 const (
 	DefaultUpdateInterval         = 30 * time.Minute
 	DefaultUpdaterInitConcurrency = 10
-	DefaultMaxConnPool            = 100
+	DefaultMaxConnPool            = 50
 )
 
 type Opts struct {
-	// the maximum size of the connection pool used by the database
+	// The maximum number of database connections in the
+	// connection pool.
 	MaxConnPool int32
-	// the connectiong string to the above data store implementation
+	// A connection string to the database Lbvuln will use.
 	ConnString string
-	// the interval in minutes which updaters will update the vulnstore
+	// An interval on which Libvuln will check for new security
+	// database updates.
 	UpdateInterval time.Duration
-	// number of updaters ran in parallel while libvuln initializes. use this to tune io/cpu on library start when using many updaters
-	UpdaterInitConcurrency int
-	// set to true to have libindex check and potentially run migrations
+	// Determines if Livuln will manage database migrations
 	Migrations bool
-	// returns the matchers to be used during libvuln runtime
-	Matchers []driver.Matcher
-	// returns the updaters to run on an interval
+	// A pointer to a slice of strings representing which
+	// updaters libvuln will create.
+	//
+	// If the pointer is nil all default UpdaterSets
+	// will be used
+	//
+	// If the pointe points to an empty array no UpdaterSets
+	// will run.
+	//
+	// The following sets are supported:
+	// "alpine"
+	// "aws"
+	// "debian"
+	// "oracle"
+	// "photon"
+	// "pyupio"
+	// "rhel"
+	// "suse"
+	// "ubuntu"
+	UpdaterSets *[]string
```

Comment:
This doesn't need to be a pointer.
```suggestion
	UpdaterSets []string
```

Reply:
@hdonnay the zero value of the struct will instantiate the array, which would be confusing if you want the zero value to be "default" which means run all updaters.

Reply:
nvm that's wrong.